core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -362,6 +364,12 @@ private[serializer] object KryoSerializer {
     classOf[StorageLevel],
     classOf[CompressedMapStatus],
     classOf[HighlyCompressedMapStatus],
+    classOf[RoaringBitmap],
+    classOf[RoaringArray],
+    classOf[RoaringArray.Element],
```

Comment:
@squito 

I think that @gssiyankai has been looking at the serialization problem through Kryo.